backend/app/api/v1/endpoints/ocm/ocmJobs.py

```diff
@@ -49,19 +51,30 @@ async def jobs(
             status_code=422,
         )
 
-    results = await getData(start_date, end_date, "ocm.elasticsearch")
+    if not offset:
+        offset = 0
 
-    if len(results) >= 1:
+    if not size:
+        size = 10000
+        offset = 0
+
+    results = await getData(start_date, end_date, size, offset, "ocm.elasticsearch")
+
+    if "data" in results and len(results["data"]) >= 1:
```

Comment:
Ah. Making the `getData` methods always return the dict rather than just `jobs` would remove this if/else complication -- adding this sort of redundancy is generally not a good idea.

Also, technically, when `results` is just a list of jobs (or, I suppose, a dataframe), you're checking whether an element is exactly `"data"`, which is likely inefficient. (Though I suppose the chances of an erroneous match aren't significant, I'd prefer to avoid that sort of dependency.)

Reply:
Yeah right. Will get this in the next PR